src/CSharp/WebAppNetCore/WebAppNetCore/Controllers/HomeController.cs

```diff
@@ -41,12 +41,18 @@ public IActionResult Index()
             ViewData["RedirectUrl"] = callbackUrl;
 
             ViewData["EnableSessionManagement"] = "No";
+            ViewData["EnablePostLogout"] = "No";
 
             if (Configuration.EnableSessionManagement())
             {
                 ViewData["EnableSessionManagement"] = "Yes";
                 ViewData["CheckSessionIframeUri"] = Configuration.CheckSessionIframeUri();
             }
+
+            if (Configuration.EnablePostLogout())
+            {
+                ViewData["EnablePostLogout"] = "Yes";
```

Comment:
ViewData["EnablePostLogout"] = Configuration.EnablePostLogout() ? "Yes" : "No" and remove the above "No" :)